frontend/packages/knative-plugin/src/topology/knative-topology-utils.ts

```diff
@@ -1246,13 +1246,15 @@ export const createSinkPubSubConnection = (
   return createEventingPubSubSink(resources.obj, targetObj);
 };
 
-export const isServerlessFunction = (element: Node): boolean => {
+export const isServerlessFunction = (element: Node | K8sResourceKind): boolean => {
```

Comment:
Better to create two separate functions or change it to this.
```suggestion
export const isServerlessFunction = (resource: K8sResourceKind): boolean => {
```

Then when you have a topology node in hand, always call it like this `isServerlessFunction(getResource(node))`

Reply:
done